.goreleaser.build.yml

```diff
@@ -56,13 +56,15 @@ builds:
 - <<: *pulumibin
   id: pulumi-language-java
   binary: pulumi-language-java
+  builder: prebuilt
   prebuilt:
-    path: goreleaser-lang/java/{{ .Os }}-{{ replace .Arch "amd64" "x64" }}/{{ .Name }}{{ .Ext }}
+    path: goreleaser-lang/java/{{ .Os }}-{{ replace .Arch "amd64" "x64" }}/{{ .Name }}
 - <<: *pulumibin
   id: pulumi-language-yaml
   binary: pulumi-language-yaml
+  builder: prebuilt
   prebuilt:
-    path: goreleaser-lang/yaml/{{ .Os }}-{{ replace .Arch "amd64" "x64" }}/{{ .Name }}{{ .Ext }}
+    path: goreleaser-lang/yaml/{{ .Os }}-{{ replace .Arch "amd64" "x64" }}/{{ .Name }}
```

Comment:
Verified locally that without `builder: prebuilt`, it was not using this key and thus not doing what was expected.